lib/manageiq/environment.rb

```diff
@@ -58,9 +60,9 @@ def self.update_ui_thread
     end
 
     def self.install_bundler(root = APP_ROOT)
-      system!("echo 'gem: --no-ri --no-rdoc --no-document' > ~/.gemrc") if ENV['CI']
-      system!("gem install bundler -v '#{bundler_version}' --conservative")
-      system!("bundle config path #{root.join('vendor/bundle').expand_path}", :chdir => root) if ENV["CI"]
+      system!("echo 'gem: --no-ri --no-rdoc --no-document' > ~/.gemrc") if ENV['TRAVIS']
+      system!("gem install bundler -v '#{bundler_version}' --conservative") unless ENV["GITHUB_ACTIONS"]
```

Comment:
Why don't we want to do this for Github Actions?

Reply:
Because there are actions that do this for us.